### PR TITLE
Rack Awareness Lab Fixes

### PR DESCRIPTION
**Pull request by chuck-confluent**
Addresses: 
- https://github.com/confluentinc/operator-earlyaccess/issues/44
- https://github.com/confluentinc/operator-earlyaccess/issues/43
- https://github.com/confluentinc/operator-earlyaccess/issues/40

Not ready for merge yet

### CODE REVIEW

**Review by arodoni — APPROVED** [review]

**Comment by chuck-confluent** [comment]
Tested in the confluent education lab portal and ready for review and merge.

@arodoni I added a commit since you approved. Could you take another quick look and merge? Thanks!

File: rackawareness/README.rst [hunk 1]
```diff
@@ -35,19 +35,19 @@ Pre-requisites: Node Labels and Service Account Rolebinding
...
-Configure your Kubernetes cluster with a service account that is configured with a clusterrole/role that provides 
-get/list access to both the pods and nodes resources.
-This is required as Kafka pods will curl kubernetes api for the node it is scheduled on using the mounted serviceAccountToken.
+#. Configure your Kubernetes cluster with a service account that is configured with a clusterrole/role that provides get/list access to both the pods and nodes resources. This is required as Kafka pods will curl kubernetes api for the node it is scheduled on using the mounted serviceAccountToken.
 
-::
+   ::
 
-  kubectl apply -f rackawareness/service-account-rolebinding.yaml
+     kubectl apply -f TUTORIAL_HOME/service-account-rolebinding.yaml
```
Comment (arodoni) [inline rackawareness/README.rst:50]: $TUTORIAL_HOME

File: rackawareness/README.rst [hunk 2]
```diff
@@ -99,4 +109,36 @@ Configure and Deploy Confluent Platform
...
+    kubectl get node \
+     -o=custom-columns=NODE:.metadata.name,ZONE:.metadata.labels."topology\.kubernetes\.io/zone" \
+       | sort -k2
+
+#. Check that the ``broker.rack`` property has been set inside broker 0.
+
+   ::
+    
+     kubectl exec -it kafka-0 -- \
+       grep 'broker.rack' confluentinc/etc/kafka/kafka.properties
```
Comment (arodoni) [inline rackawareness/README.rst:135]: Missing a pipe?

File: rackawareness/README.rst (hunk 1 above)
Comment (chuck-confluent) [reply rackawareness/README.rst]: good call! Fixed in commit 86ab66f

File: rackawareness/README.rst (hunk 2 above)
Comment (chuck-confluent) [reply rackawareness/README.rst:135]: No, this is how kubectl exec works. From https://kubernetes.io/docs/tasks/debug-application-cluster/get-shell-running-container/:

> Note: The double dash (--) separates the arguments you want to pass to the command from the kubectl arguments.

This command worked as-is when I tested it

**Comment by chuck-confluent** [comment]
@rohit2b could you give this a quick glance and merge if you think it's ok? Alexandra gave it a look already, and I tested in the lab platform. I would merge it myself, but I've been bitten before :-)

File: rackawareness/confluent-platform.yaml [hunk 3]
```diff
@@ -21,7 +21,7 @@ spec:
   replicas: 6
   rackAssignment:
     nodeLabels:
-    - failure-domain.beta.kubernetes.io/zone
+    - topology.kubernetes.io/zone
```
Comment (rohit2b) [inline rackawareness/confluent-platform.yaml:24]: ahh I see this:
https://kubernetes.io/docs/reference/labels-annotations-taints/#failure-domainbetakubernetesioregion

Starting in v1.17, the label `failure-domain.beta.kubernetes.io/zone` is deprecated in favor of topology.kubernetes.io/zone.

I see on my test GKE cluster that both label names are available.

If we want to support K8s versions 1.15 -> 1.20, which one label name should we specify? @xnulinu @andrewxding

Comment (chuck-confluent) [reply rackawareness/confluent-platform.yaml:24]: Maybe we should use the not deprecated one and make a note for k8s version less than 1.17 to use the other?

Comment (chuck-confluent) [reply rackawareness/confluent-platform.yaml:24]: Made note in commit  ba6a051

**Review by rohit2b — APPROVED** [review]
LGTM!

**Pull request merged**